### DISCUSSION TITLE
Display animations from compressed images or expand storage?

### DISCUSSION THREAD

**Discussion by MintKind** [Bodmer/TFT_eSPI#2391] [Q&A - General] [answered]
I am currently working with a ESP8266 and circular 240x240 GC9A01 display. I want to display pixel art animation with total more than 300 frames combined. What would be the best solution to achieve this?
If I convert each 240x240 frame into the .h format I get 512kb per file. 512 times the 300 frames is not store able on the ESPs memory alone. Would it be possible to use the PNGs unconverted?  Than it would easy fit on the ESPs memory.
Or would I need to connect an SD card for example and store the converted images on that.
But than I don't know if either of these solutions would be performent enough to run a 30 frames per sec animations.
Does anyone have a opinion or solution on this, would be very helpful :)

**Comment by Bodmer** [comment, marked as answer]
The ESP8266 will only be able to achieve a few frames per second so it will not be possible to do what you wish to.

**Reply by MintKind** [reply]
> The ESP8266 will only ne able to achieve a few frames per second so it will not be possible to do what you wish to.
Is there an alternative chip in a similar form-factor that could achieve these results?
